README.md

```diff
@@ -45,21 +45,21 @@ cd physlr/src && make install PREFIX=/opt/physlr
 ## Generating Physlr Physical Map with stLFR reads
 
 To construct a physical map, you need linked reads from 10X Genomics or stLFR. In addition, to visualize the correctness and contiguity of the physical map, you need a reference genome.
-In this example, the linked reads and reference genome are called `linkedreads.fq.gz` and `reference.fa`, respectively. The linked reads are from stLFR so we specify `minimizer_overlap=stLFR` to use the default value for stLFR reads.
+In this example, the linked reads and reference genome are called `linkedreads.fq.gz` and `reference.fa`, respectively. The linked reads are from stLFR so we specify `protocol=stlfr` to use the default value for stLFR reads.
 
 ```
 cd experiment
-bin/physlr-make physical-map lr=linkedreads ref=reference minimizer_overlap=stLFR
+bin/physlr-make physical-map lr=linkedreads ref=ref protocol=stlfr
```

Comment:
Not sure when we made this change in the master from `ref `to `reference` and introduced this conflict but this should be `ref=reference`.
I rebased and it showed up as a conflict and I merged @lcoombe's version and now I am suggesting this change :D

Reply:
I think that's fine! That's a strange conflict haha

Reply:
As in other parts of the readme, it is referred to as `reference` I think we still need to make this change. I wanted to do so but I thought it would be better to let the author know.